tests/e2e/cluster/up.sh

```diff
@@ -58,7 +58,7 @@ main() {
 	# Untaint the node so that pods can be scheduled on it.
 	for role in master control-plane; do
 		kubectl taint nodes "$(hostname)" \
-			"node-role.kubernetes.io/$role:NoSchedule-"
+			"node.kubernetes.io/$role:NoSchedule-"
```

Comment:
@kartikjoshi21 - the tests are getting an error:
```
error: taint "node.kubernetes.io/master:NoSchedule" not found
```
which I think is related to this change?

Reply:
This is how I understand the situation:

* the operator CI deploys the test cluster with kubeadm, which labels the node as  `node-role.kubernetes.io`. I will need to change the kubeadm configs to use the new label;
* the operator CI for enclave-cc deploys the test cluster with kind, which already labels the node as `node.kubernetes.io`. Additionally the workflow script add the `node-role.kubernetes.io` label, that explains why the enclave-cc job didn't fail (?)

This PR fixes #194 that is a duplicated of https://github.com/confidential-containers/operator/issues/130 . On this last issue there is some discussion whether change or not the label. I am in favor of the change btw, but as we are about to release 0.5 and it has potential to break stuffs: @kartikjoshi21 mind if we post-pone this change to after the release? If yes then I will put the "do-not-merge" label on this PR.

Reply:
@wainersm  I think we can postpone it, and CAA one which is related to this one as well. Meanwhile i have a workaround to label node to node-role.* which is already merged.